recipe/build.sh

```diff
@@ -15,11 +20,34 @@ cd build
 cmake ${CMAKE_ARGS} \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
   -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_LIBRARY_PATH="${PREFIX}" \
+  -DLLVM_ENABLE_RTTI=ON \
+  -DLLVM_EXTERNAL_LIT="${BUILD_PREFIX}/bin/llvm-lit" \
+  -DMLIR_INCLUDE_DOCS=OFF \
+  -DMLIR_INCLUDE_TESTS=ON \
   -DLLVM_BUILD_LLVM_DYLIB=ON \
+  -DMLIR_INCLUDE_INTEGRATION_TESTS=ON \
+  -DLLVM_SHLIB_OUTPUT_INTDIR="${SRC_DIR}/build/lib" \
```

Comment:
Should we add a comment here considering we spent a couple of hours to figure this one out?

Reply:
done